docs/configuredgraphfactory.adoc

```diff
@@ -403,7 +403,7 @@ def g3 = ConfiguredGraphFactory.create("graph3"); //storage directory === /data/
 
 [source, gremlin]
 ----
-gremlin> :remote connect tinkerpop.server conf/remote.yaml
+gremlin> :remote connect tinkerpop.server conf/remote.yaml session
```

Comment:
Add some text before the code block to highlight why a session is being used on the remote connection. Add a link reference to the TinkerPop docs on it. https://tinkerpop.apache.org/docs/current/reference/#console-sessions

Reply:
@pluradj The details on why sessions add value is in #974 and I think this would be a bit redundant for the ConfiguredGraphFactory doc. Are you sure you want it added her as well?

Reply:
I think text with a link to the docs in #974 would be sufficient then.